scootapi/testHelpers/generators.go

```diff
@@ -0,0 +1,51 @@
+package testHelpers
+
+import (
+	"github.com/scootdev/scoot/scootapi/gen-go/scoot"
+	"math/rand"
+)
+
+// Test Helpers that are useful for Generating random Scoot Api Structs
+// To help with testing the Scoot API and Scheduler
+
+// Generates a scoot.JobDefinition
+func GenJobDefinition(rng *rand.Rand) *scoot.JobDefinition {
+
+	def := scoot.NewJobDefinition()
+	def.Tasks = make(map[string]*scoot.TaskDefinition)
+
+	numTasks := rng.Intn(100) + 1
+	for i := 0; i < numTasks; i++ {
+		taskId := GenTaskId(rng)
```

Comment:
So, it'll be an error if these end up colliding. So maybe just have 

taskId := fmt.Sprintf("task%d", i) ?